dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.idl

```diff
@@ -5,9 +5,56 @@ import "..\AppNotifications\AppNotifications.idl";
 
 namespace Microsoft.Windows.AppNotifications.Builder
 {
-    [contractversion(1)]
+    [contractversion(2)]
     apicontract AppNotificationBuilderContract {}
 
+    [contract(AppNotificationBuilderContract, 2)]
+    runtimeclass AppNotificationHeader
+    {
+        AppNotificationHeader();
+
+        String Id;
+        String Title;
+        Windows.Foundation.Collections.IMap<String, String> Arguments;
+
+        AppNotificationHeader(String id, String title);
+
+        AppNotificationHeader AddArgument(String key, String value);
+    };
+
+    [contract(AppNotificationBuilderContract, 2)]
+    enum AppNotificationTextStyle
+    {
+        Default,
+        Caption,
+        CaptionSubtle,
+        Body,
+        BodySubtle,
+        Base,
+        BaseSubtle,
+        Subtitle,
+        SubtitleSubtle,
+        Title,
+        TitleSubtle,
+        TitleNumeral,
+        Subheader,
+        SubheaderSubtle,
+        SubheaderNumeral,
+        Header,
+        HeaderSubtle,
+        HeaderNumeral,
+    };
+
+    [contract(AppNotificationBuilderContract, 2)]
+    enum AppNotificationTextAlign
+    {
+        Default,
+        Auto,
```

Comment:
What's the difference between Default and Auto?

Reply:
Default ==> Default value. Alignment is automatically determined by the renderer.
Auto ==> Alignment determined by the current language and culture.